src/lib_ccx/ccx_encoders_srt.c

```diff
@@ -83,6 +83,7 @@ int write_stringz_as_srt(char *string, struct encoder_ctx *context, LLONG ms_sta
 
 int write_cc_bitmap_as_srt(struct cc_subtitle *sub, struct encoder_ctx *context)
 {
+    mprint("\nNow subtitle %x is really written into file\n", sub);
```

Comment:
Debug messages should use the debug function, not mprint()...

Reply:
Sorry, it's supposed to be temporary but I forgot to remove them.